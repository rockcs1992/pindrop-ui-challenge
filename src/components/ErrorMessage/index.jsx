import React from 'react';
import styled from 'styled-components';

const Error = styled.h3`
    display: inline;
    color: red;
`;

const Center = styled.div`
    text-align: center;
`;

const ErrorMessage = () =>
    <Center>
        <Error>Something went wrong</Error>
    </Center>;

export default ErrorMessage;