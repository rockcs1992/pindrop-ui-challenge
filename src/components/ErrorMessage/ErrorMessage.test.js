import React from 'react';
import { render } from 'enzyme';
import ErrorMessage from './index';

describe('Error Message', () => {
    it('should render correctly', () => {
        const tree = render(<ErrorMessage />);
        expect(tree).toMatchSnapshot();
    })
})