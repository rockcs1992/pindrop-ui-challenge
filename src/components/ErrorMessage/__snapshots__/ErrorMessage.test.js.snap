// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Error Message should render correctly 1`] = `
<div
  class="sc-bwzfXH ewMBHc"
>
  <h3
    class="sc-bdVaJa bXaXgR"
  >
    Something went wrong
  </h3>
</div>
`;
